types/DuplicateRecordError.go

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) HashiCorp, Inc.
```

Comment:
Nit: we don't use camelcase file names, this should be `duplicate_record_error.go`, or simply call it `errors.go`.

Reply:
d'oh